autoload/neomake/makers/cargo.vim

```diff
@@ -1,11 +1,16 @@
 " vim: ts=4 sw=4 et
 
-function! neomake#makers#cargo#cargo()
+function! neomake#makers#cargo#cargo() abort
     return {
         \ 'args': ['build'],
         \ 'errorformat':
-            \   '%-Z%f:%l,' .
-            \   '%+C %s,' .
-            \   '%A%f:%l:%c: %*\d:%*\d\ %t%*[^:]: %m,',
+            \ '%Eerror[E%n]: %m,'.
+            \ '%Eerror: %m,'.
+            \ '%Wwarning: %m,'.
+            \ '%Inote: %m,'.
+            \ '%-Z\ %#-->\ %f:%l:%c,'.
```

Comment:
@YorickPeterse i am aware of that, but the only usage of that flag in this PR is in the `rust.vim` file. I would like to know why did you comment on `cargo.vim` file. Is there something i am missing here?